modules/custom/movie_reservations/src/Controller/MovieController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\movie_reservations\Controller\MovieController.
+ */
+namespace Drupal\movie_reservations\Controller;
+class MovieController {
+  public function content() {
+    return array(
+      '#type' => 'markup',
+      '#markup' => ('List of available movies: '),
+    );
+  }
+}
```

Comment:
Ovde je sve zbijeno, pa mozes koristiti od editora opciju koja formatira kod i sam ubacuje redove za preglednos, ili da sam po sebi negde ubacis prazan red. Necu sada traziti ponovo izmene za to, ali obrati paznju u sklopu sledecih taskova.